FoxHound.App.Tests/Blogs/CreateBlog/CreateBlogCommandValidatorTests.cs

```diff
@@ -0,0 +1,38 @@
+using AutoFixture;
+using AutoFixture.AutoMoq;
+using FluentValidation.TestHelper;
+using FoxHound.App.Blogs.CreateBlog;
+using FoxHound.App.Data;
+using Microsoft.EntityFrameworkCore;
+using System;
+using Xunit;
+
+namespace FoxHound.App.Tests.Blogs.CreateBlog
+{
+    public class CreateBlogCommandValidatorTests
+    {
+        private readonly IFixture _fixture;
+        private readonly IFoxHoundData _foxHoundData;
+        private readonly DbContextOptions<FoxHoundData> _dbContextOptions;
+
+        public CreateBlogCommandValidatorTests()
+        {
+            _fixture = new Fixture();
+            _fixture.Customize(new AutoMoqCustomization());
+
+            _dbContextOptions = new DbContextOptionsBuilder<FoxHoundData>().UseInMemoryDatabase(Guid.NewGuid().ToString()).Options;
+            _foxHoundData = new FoxHoundData(_dbContextOptions);
+            _fixture.Inject(_foxHoundData);
```

Comment:
These aren't needed at the moment, but if we wanted to test for something like an owner can only have 1 blog (which probably makes sense as a validation), then this would be needed in order to test what's already in the db.

Reply:
I was getting a little ahead of myself...agree I will remove.